README.md

```diff
@@ -59,7 +65,7 @@ If you want to customize the look and feel of your admin, you should use the SAS
 
 ```css
 @import "admin_variables";
-@import "../../../node_modules/torch/assets/css/torch";
+@import "~torch/assets/css/index";
```

Comment:
This is the source `.sass` file and I didn't want it to get confused with the built `torch.css`. I think `app.sass` and `app.js` might be better names for the source files, that seems to be our standard for `phx` projects. Then we can call our compiled assets `torch.css` and `torch.js`. We seem to be following `phx` folder structure, so `phx` file naming would be good for consistency sake. What do you think @zberkom?